articles/install-guide/python.md

```diff
@@ -13,25 +13,31 @@ The [qsharp package for Python](https://pypi.org/project/qsharp/) provides inter
 
 ## Pre-requisites ##
 
-To get started with the `qsharp` package, you'll need the following prerequisites:
+Before installing the `qsharp` package, you'll need the following pre-requisites:
 - IQ#.
 - Python 3.6 or later,
 
-Make sure you have followed the installation steps in the [getting started guide](xref:microsoft.quantum.install) for IQ#.
+IQ# provides the core functionality of compiling and simulating Q# operations.
```

Comment:
I don't have the context for this change, but it looks a bit weird - we point people here from install guide, and this part of the instructions is duplicated. Should we somehow point it out? (+ the same comment about 10 minutes applies)